<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapMainWindow</class>
 <widget class="QMainWindow" name="MapMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="localArea" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="countryLabel">
         <property name="text">
          <string>Country:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cityLabel">
         <property name="text">
          <string>City:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="latitudeLabel">
         <property name="text">
          <string>Latitude:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="longitudeLabel">
         <property name="text">
          <string>Longitude:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="displayArea" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="inputArea" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="routeArea" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QGridLayout" name="routeInput">
            <item row="0" column="1">
             <widget class="QPushButton" name="route1">
              <property name="text">
               <string>Route2</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="route0">
              <property name="text">
               <string>Route1</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="route3">
              <property name="text">
               <string>Route4</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QPushButton" name="route4">
              <property name="text">
               <string>Route5</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="route5">
              <property name="text">
               <string>Route6</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="route2">
              <property name="text">
               <string>Route3</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="route7">
              <property name="text">
               <string>Route8</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QPushButton" name="route8">
              <property name="text">
               <string>Route9</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="route6">
              <property name="text">
               <string>Route7</string>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QPushButton" name="route9">
              <property name="text">
               <string>Route10</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="sugRouteBtn">
              <property name="text">
               <string>   Perferred Route   </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="allroute">
              <property name="text">
               <string>    AllRoute    </string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="cfmRouteBtn">
              <property name="text">
               <string>&amp;Confirm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cclRouteBtn">
              <property name="text">
               <string>&amp;Cancel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="backArea" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>480</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="backBtn">
            <property name="text">
             <string>&amp;Back</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="placeArea" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QFormLayout" name="placeInput">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Origin:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="originInput"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Destination:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="destInput"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="cfmPlaceBtn">
              <property name="text">
               <string>&amp;Confirm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clrPlaceBtn">
              <property name="text">
               <string>&amp;Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Menu">
    <property name="title">
     <string>&amp;Menu</string>
    </property>
   </widget>
   <addaction name="menu_Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clrPlaceBtn</sender>
   <signal>clicked()</signal>
   <receiver>originInput</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>714</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>658</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clrPlaceBtn</sender>
   <signal>clicked()</signal>
   <receiver>destInput</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>714</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>684</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
